test/web3_test.erl

```diff
@@ -4,8 +4,8 @@
 -include_lib("stdlib/include/assert.hrl").
 
 web3_test() ->
-  {ok, Pid} = gen_server:start_link(web3, [], []),
-  ?debugFmt("~p~n",[Pid]),
+  {ok, Pid} = gen_server:start(web3, [], []),
```

Comment:
whats the difference between start and start_link?

Reply:
start_link links the two actors so if one crashes so does the other.... Which means when I stop the web3 gen_server it will crash the eunit process if I use start_link D: